club/models.py

```diff
@@ -0,0 +1,88 @@
+from django.contrib.auth.models import AbstractUser
+from django.db import models
+
+
+class Member(AbstractUser):
+    TYPE_CHOICES = [('Admin', 'Admin'),
```

Comment:
nice you could also store them as ints, e.g. https://stackoverflow.com/a/74191713